src/librados/IoCtxImpl.h

```diff
@@ -206,6 +206,7 @@ struct librados::IoCtxImpl {
   int aio_writesame(const object_t &oid, AioCompletionImpl *c,
 		    const bufferlist& bl, size_t write_len, uint64_t off);
   int aio_remove(const object_t &oid, AioCompletionImpl *c);
+  int aio_remove(const object_t &oid, AioCompletionImpl *c, int flags);
```

Comment:
@jdurgin adding a new overload breaks the ABI, right?  this should be aio_remove2?

Should we just use aio_operate from the cli instead?